onnxscript/rewriter/onnxruntime/transformers/biassplitgelu.py

```diff
@@ -29,4 +29,4 @@ def ggelu(input, weight, bias):
             return msft_opset.BiasSplitGelu(matmul_input, bias)
 
         function_proto = onnxscript.script(default_opset=op)(ggelu).to_function_proto()  # type: ignore[arg-type]
```

Comment:
Here for example we are returning protos. Would it make sense to return an ir function?

Reply:
I see. That makes sense. ir.Function goes in, and we should return the same type.

Reply:
Done